### DISCUSSION TITLE
Can not add integration

### DISCUSSION THREAD

**Discussion by majorcs** [wlcrs/huawei_solar#263] [Setting up the integration] [answered]
I'm trying to add this integration to HassIO, using HACS, and it downloads it without any problem. However, when I try to add the integration, I get this error:
![image](https://user-images.githubusercontent.com/17719849/213527453-91b6af1e-5a34-4b20-bdb1-077eab099679.png)

In the logs, I can see this at the same time:
`2023-01-19 19:18:35.805 ERROR (MainThread) [homeassistant.config_entries] Error occurred loading configuration flow for integration huawei_solar: cannot import name 'AsyncModbusSerialClient' from 'pymodbus.client' (/usr/local/lib/python3.10/site-packages/pymodbus/client/__init__.py)`

I think it is a HassIO related issue, but everything is managed by HassIO, so I don't know how to fix this. 
Any idea what can go wrong?
(Using the latest version from all components at the moment)

**Comment by wlcrs** [comment, marked as answer]
Are you using other integrations that use Modbus? Does v1.1.6 of the integration work for you?

**Reply by majorcs** [reply]
Thanks for the hint, I found a similar problem with another integration (looks like old modbus library in the HA):
https://github.com/WillCodeForCats/solaredge-modbus-multi/issues/162

With the 1.1.6 it seems better, now I'm trying to configure this.

Thanks for the fast reply!

**Reply by wlcrs** [reply]
Will be fixed in HA2023.2 thanks to https://github.com/home-assistant/core/pull/85961
